packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -137,44 +137,55 @@ export class OdspDeltaStorageWithCache implements IDocumentDeltaStorageService {
         let opsFromCache = 0;
         let opsFromStorage = 0;
 
-        const stream = requestOps(
```

Comment:
Note: there are no changes here (code moved around and indentation change). Other than added asserts on lines 183+.

Reply:
Not true any more - added this.validateMessages() calls in each of the forks